### DISCUSSION TITLE
Not able to connect Github account

### DISCUSSION THREAD

**Discussion by andrealemonroy** [community/community#41877] [Copilot Conversations] [answered]
### Select Topic Area

Product Feedback

### Body

It throws this error:

Macbook Air M1, it was working well, but accidentally I clicked not log in and then starts with that error.

GitHub Copilot could not connect to server. Extension activation failed: "Timed out waiting for authentication provider to register

Please, help

Output: [ERROR] [default] [2022-12-15T23:08:40.829Z] GitHub Copilot could not connect to server. Extension activation failed: "Timed out waiting for authentication provider to register"
<img width="1376" alt="Captura de Pantalla 2022-12-15 a la(s) 18 09 20" src="https://user-images.githubusercontent.com/39166187/207986005-604179a3-e519-43dd-84f7-8f68e3d4656f.png">

**Comment by andrealemonroy** [comment, marked as answer]
It take some time, but I was able to fixed, as many posts says, need to delete vs code completely. In mac with this commands:

rm -fr ~/Library/Preferences/com.microsoft.VSCode.helper.plist 
rm -fr ~/Library/Preferences/com.microsoft.VSCode.plist 
rm -fr ~/Library/Caches/com.microsoft.VSCode
rm -fr ~/Library/Caches/com.microsoft.VSCode.ShipIt/
rm -fr ~/Library/Application\ Support/Code/
rm -fr ~/Library/Saved\ Application\ State/com.microsoft.VSCode.savedState/
rm -fr ~/.vscode/

rm -rf ~/.vscode*

Founded here: https://stackoverflow.com/questions/42603103/how-to-completely-uninstall-vs-code-on-mac

**Comment by akhil9tiet** [comment]
What is Windows issue resolution

**Reply by akhil9tiet** [reply]
I intsalled vs code completely and reinstalled it but wont still work

**Reply by davecheney** [reply]
<img width="475" alt="Screen Shot 2021-11-30 at 8 03 57 am" src="https://user-images.githubusercontent.com/7171/143942796-0c2b6d34-e0f4-46fe-b349-f26087b2c5ed.png">
Please try signing out, copilot will prompt you to sign back in.

**Comment by onlyonedats** [comment]
Find the solution.

Remove the latest version of VS Code from Macbook

Install an older version (1.93.1)

Update it later through the app itself: its works well.
.....

find the URL to install through terminal.

curl -L "https://update.code.visualstudio.com/1.93.1/darwin-universal/stable" -o VSCode-darwin.zip &&
unzip VSCode-darwin.zip -d /Applications

Enjoy 😊 😉
